js-miniapp-bridge/src/types/token-data.ts

```diff
@@ -0,0 +1,5 @@
+/** Token data type. */
+export interface TokenData {
```

Comment:
Maybe the name `AccessTokenData` would be better to match the function name `getAccessToken`? 

Also, we may have other types of tokens in the future, such as an ID Token (Open ID Connect), so it's good to be specific on the token type.

Reply:
I have updated to `AccessTokenData` 👍